can/tests/test_messages.cpp

```diff
@@ -49,8 +49,9 @@ SCENARIO("message serializing works") {
         auto message = MoveCompleted{.group_id = 1,
                                      .seq_id = 2,
                                      .current_position_um = 0x3456789a,
+                                     .encoder_position = 0x05803931,
                                      .ack_id = 1};
-        auto arr = std::array<uint8_t, 8>{};
+        auto arr = std::array<uint8_t, 11>{};
```

Comment:
```suggestion
        auto arr = std::array<uint8_t, 12>{};
```

Adding the extra byte here implicitly tests that the serialization only writes the necessary data even if there is more available space in the buffer.